README.md

```diff
@@ -73,6 +73,38 @@ _Additional information_
 
 ### Usage notes
 
+**Zero-Downtime Tables**
+Starting from adapter version `1.0.3` there is a way to do `dbt run` without downtime on table update  
+It works in the following way:  
+1. Enable zero-downtime for tables   
+a) Add var `table_zero_downtime` to the `dbt_project.yml`  
+```
+vars:
+  table_zero_downtime: true
+```
+b) Alternatively add tag `table_zero_downtime` to the specific model with `table` materialization  
+```
+{{config(tags=['table_zero_downtime'])}}
+```
+2. Create a companion view for your table  
+a) It's usually is very simple like `select * from {{ref('table_model')}}`  
```

Comment:
Curios question: why not let the code do this themselves? Like create the real table with a temp name and the name which corresponds to the filename as a view with this simple select star?

Reply:
@jankatins Good catch actually - Thanks! Haven't considered such an option to be honest. Will think on it - since it might eliminate several downsides of the solution